### PR TITLE
Crop, Livestock, and Property Resource Implementations.

### PR DESCRIPTION
**Pull request by nickwxo**
This pull request implements the Crop, Livestock, and Property endpoints as well as the ACL files for them. They are all properly authenticated by token.

### CODE REVIEW

File: routes/resources/crop/crop.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    connection1.on('error', function(err) {
+        if(err) {
+            return next(err);
+        }
+    });
+
+};
+
+
+exports.getCropByID = function(req, res, next) {
```
Comment (matjames007) [inline routes/resources/crop/crop.js:139]: @nick22891 implementation needed.

File: routes/resources/property/router-property.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+var express = require('express');
+var Property = require('./property');
+var router = express.Router();
+
+/**
+ * End Points relevant to Farmers
+ */
+router.get('/properties', Property.getAllProperties);
+router.get('/property/:id', Property.getPropertyByID);
```
Comment (matjames007) [inline routes/resources/property/router-property.js:13]: @nick22891 implement all the `getXByID` functions.

**Review by matjames007 — DISMISSED** [review]
Still need to implement a few of more of the ACL + resources endpoints.

**Comment by nickwxo** [comment]
@matjames007 This pull request can be reviewed again.

**Comment by nickwxo** [comment]
This PR can now be reviewed again.

**Comment by nickwxo** [comment]
Actually no - just remembered that the logging hasn't been implemented for the new (non-farmer endpoints using the Sequelize library). Will implement that and update again this evening.

**Comment by paradoxxjm** [comment]
@nick22891 what's the status of the pull request?

**Review by matjames007 — APPROVED** [review]
No issues found during review.

**Comment by matjames007** [comment]
@nick22891 may need merge `develop` before I can proceed.

**Comment by nickwxo** [comment]
@matjames007 Conflicts resolved.

**Pull request merged**